src/helpers/amoc_xmpp.erl

```diff
@@ -8,8 +8,12 @@
 -export([bucket_neighbours/2]).
 
 -include_lib("kernel/include/logger.hrl").
+-define(V(X), (fun amoc_config_validation:X/1)).
 
--required_variable([#{name => xmpp_servers, description => "the list of XMPP servers"}]).
+-required_variable([
+    #{name => xmpp_servers, description => "the list of XMPP servers"},
+    #{name => legacy_tls, description => "use legacy tls connection",
+      default_value => false,  verification => ?V(boolean)}]).
```

Comment:
Minor: extra space. I have to ask though, why not `verification = fun is_boolean/1` like in `amoc_xmpp_handlers`?